package.json

```diff
@@ -6,9 +6,10 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "gsap": "^3.11.4",
```

Comment:
GSAP may not ever end up getting used. Make sure to npm uninstall gsap if you don't use it in any future components. It was originally being used for the letter rendering, which may or may not be used in the future.